### PR TITLE
PLATFORM-4806 | Change all references to VSTF in code to SOAP

### PR DESCRIPTION
**Pull request by maaartyyynaa**
Messages to be translated:
- staffpowers-ipblock-abort

### CODE REVIEW

File: extensions/GlobalMessages/GlobalMessagesR.i18n.php [hunk 1]
```diff
@@ -34516,7 +34516,7 @@ function, o le version specificate non existe, o tu essaya celar le version actu
...
 |-
 ! style="text-align:right;white-space:nowrap;vertical-align:text-top;" | Użytkownicy :
-| [[Special:ListUsers/sysop|{{int:group-sysop}}]] <b>&middot;</b> [[Special:ListUsers/bureaucrat|{{int:group-bureaucrat}}]] <b>&middot;</b> [[Special:ListGlobalUsers/helper|{{int:group-helper}}]] <b>&middot;</b> [[Special:ListGlobalUsers/vstf|{{int:group-vstf}}]] <b>&middot;</b> [[Special:ListGlobalUsers/staff|{{int:group-staff}}]] <b>&middot;</b> <small>[[Special:ListUsers|wszyscy...]]</small>
+| [[Special:ListUsers/sysop|{{int:group-sysop}}]] <b>&middot;</b> [[Special:ListUsers/bureaucrat|{{int:group-bureaucrat}}]] <b>&middot;</b> [[Special:ListGlobalUsers/helper|{{int:group-helper}}]] <b>&middot;</b> [[Special:ListGlobalUsers/soap|{{int:group-soad}}]] <b>&middot;</b> [[Special:ListGlobalUsers/staff|{{int:group-staff}}]] <b>&middot;</b> <small>[[Special:ListUsers|wszyscy...]]</small>
```
Comment (Markus-Rost) [inline extensions/GlobalMessages/GlobalMessagesR.i18n.php:34519]:
```suggestion
| [[Special:ListUsers/sysop|{{int:group-sysop}}]] <b>&middot;</b> [[Special:ListUsers/bureaucrat|{{int:group-bureaucrat}}]] <b>&middot;</b> [[Special:ListGlobalUsers/helper|{{int:group-helper}}]] <b>&middot;</b> [[Special:ListGlobalUsers/soap|{{int:group-soap}}]] <b>&middot;</b> [[Special:ListGlobalUsers/staff|{{int:group-staff}}]] <b>&middot;</b> <small>[[Special:ListUsers|wszyscy...]]</small>
```

File: extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php [hunk 2]
```diff
@@ -2971,7 +2971,7 @@
 	'user-identity-box-group-staff' => 'Сотрудник',
 	'user-identity-box-group-wiki-manager' => 'Руководитель вики',
 	'user-identity-box-group-helper' => 'Помощник',
-	'user-identity-box-group-vstf' => 'VSTF',
+	'user-identity-box-group-soad' => 'SOAD',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php]:
```suggestion
	'user-identity-box-group-soap' => 'SOAP',
```

File: extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php [hunk 3]
```diff
@@ -3128,7 +3128,7 @@
 	'user-identity-box-group-founder' => 'Grundare',
 	'user-identity-box-group-staff' => 'Personal',
 	'user-identity-box-group-helper' => 'Hjälpare',
-	'user-identity-box-group-vstf' => 'Frivillig spambekämpningsstyrka',
+	'user-identity-box-group-soad' => 'Frivillig spambekämpningsstyrka',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php]:
```suggestion
	'user-identity-box-group-soap' => 'Frivillig spambekämpningsstyrka',
```

File: extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php [hunk 4]
```diff
@@ -3258,7 +3258,7 @@
 	'user-identity-box-group-founder' => 'Tagapagtatag',
 	'user-identity-box-group-staff' => 'Kawani',
 	'user-identity-box-group-helper' => 'Tagatulong',
-	'user-identity-box-group-vstf' => 'Kusang-loob na Hukbo na Panglaban sa Basurang Liham',
+	'user-identity-box-group-soad' => 'Kusang-loob na Hukbo na Panglaban sa Basurang Liham',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php]:
```suggestion
	'user-identity-box-group-soap' => 'Kusang-loob na Hukbo na Panglaban sa Basurang Liham',
```

File: extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php [hunk 5]
```diff
@@ -3486,7 +3486,7 @@
 	'user-identity-box-group-founder' => 'Sáng lập viên',
 	'user-identity-box-group-staff' => 'Nhân viên',
 	'user-identity-box-group-helper' => 'Hỗ trợ viên',
-	'user-identity-box-group-vstf' => 'VSTF',
+	'user-identity-box-group-soad' => 'SOAD',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php]:
```suggestion
	'user-identity-box-group-soap' => 'SOAP',
```

File: extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php [hunk 6]
```diff
@@ -3598,7 +3598,7 @@
 	'user-identity-box-group-staff' => '职员',
 	'user-identity-box-group-wiki-manager' => 'Wiki主管',
 	'user-identity-box-group-helper' => '助手',
-	'user-identity-box-group-vstf' => 'VSTF',
+	'user-identity-box-group-soad' => 'SOAD',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php]:
```suggestion
	'user-identity-box-group-soap' => 'SOAP',
```

File: extensions/wikia/UserProfilePageV3/UserTagsStrategy.class.php [hunk 7]
```diff
@@ -18,7 +18,7 @@ class UserTagsStrategy extends WikiaObject {
 		'wiki-manager',
 		'content-team-member',
 		'helper',
-		'vstf',
+		'soad',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserTagsStrategy.class.php]:
```suggestion
		'soap',
```

File: extensions/wikia/WikiaIrcGateway/WikiaIrcGateway.i18n.php [hunk 8]
```diff
@@ -110,7 +110,7 @@
 	'ircgate-channellist' => '*group: General
 * #wikia-es
 * ##wikia
-* #wikia-vstf
+* #wikia-soad
```
Comment (Markus-Rost) [inline extensions/wikia/WikiaIrcGateway/WikiaIrcGateway.i18n.php]:
```suggestion
* #wikia-soap
```

File: includes/Block.php [hunk 9]
```diff
@@ -1229,7 +1229,7 @@ public function setBlockEmail( $blockEmail ) {
 
 	/**
 	 * Wikia change
-	 * SUS-288: Hide blocker name from logs and error pages if the block was made by staff/VSTF
+	 * SUS-288: Hide blocker name from logs and error pages if the block was made by staff/SOAD
```
Comment (Markus-Rost) [inline includes/Block.php]:
```suggestion
	 * SUS-288: Hide blocker name from logs and error pages if the block was made by staff/SOAP
```

File: includes/Block.php [hunk 10]
```diff
@@ -1245,7 +1245,7 @@ public function shouldHideBlockerName() {
 
 	/**
 	 * SUS-288: Return the group name that should be shown instead of user name if the blocker name is hidden
-	 * (i.e. if the block was made by staff/VSTF)
+	 * (i.e. if the block was made by staff/SOAD)
```
Comment (Markus-Rost) [inline includes/Block.php]:
```suggestion
	 * (i.e. if the block was made by staff/SOAP)
```

File: includes/api/ApiQueryUsers.php [hunk 11]
```diff
@@ -233,7 +233,7 @@ public function execute() {
 
 	/**
 	 * SUS-1472: Check if current request is authorized to view global block information
-	 * (it is internal request or request by staff/VSTF/helper user)
+	 * (it is internal request or request by staff/SOAD/helper user)
```
Comment (Markus-Rost) [inline includes/api/ApiQueryUsers.php]:
```suggestion
	 * (it is internal request or request by staff/SOAP/helper user)
```

File: includes/api/wikia/ApiQueryUserGroupMembers.php [hunk 12]
```diff
@@ -86,7 +86,7 @@ protected function getParamDescription() {
 
 	public function getExamples() {
 		return [
-			'api.php?action=query&list=groupmembers&gmgroups=vstf|staff&gmlimit=15',
+			'api.php?action=query&list=groupmembers&gmgroups=soad|staff&gmlimit=15',
```
Comment (Markus-Rost) [inline includes/api/wikia/ApiQueryUserGroupMembers.php]:
```suggestion
			'api.php?action=query&list=groupmembers&gmgroups=soap|staff&gmlimit=15',
```

File: includes/wikia/Wikia.php [hunk 13]
```diff
@@ -1774,7 +1774,7 @@ static function canEditInterfaceWhitelist(
 			return true;
 		}
 
-		// Allow trusted users to edit interface messages (util, vstf, select admins)
+		// Allow trusted users to edit interface messages (util, soad, select admins)
```
Comment (Markus-Rost) [inline includes/wikia/Wikia.php]:
```suggestion
		// Allow trusted users to edit interface messages (util, soap, select admins)
```

File: includes/wikia/tests/HideBlockerNameTest.php [hunk 14]
```diff
@@ -24,7 +24,7 @@ public function testHideBlockerName(
 	public function hideBlockerNameDataProvider() {
 		yield 'user name hidden for staff' => [ 1, true ];
 		yield 'user name hidden for helpers' => [ 2, true ];
-		yield 'user name hidden for vstf' => [ 3, true ];
+		yield 'user name hidden for soad' => [ 3, true ];
```
Comment (Markus-Rost) [inline includes/wikia/tests/HideBlockerNameTest.php]:
```suggestion
		yield 'user name hidden for soap' => [ 3, true ];
```

File: lib/Wikia/src/Service/User/Permissions/PermissionsConfiguration.php [hunk 15]
```diff
@@ -41,7 +41,7 @@ class PermissionsConfiguration {
 		'util',
 		'vanguard',
 		'voldev',
-		'vstf',
+		'soad',
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/PermissionsConfiguration.php]:
```suggestion
		'soap',
```

File: lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php [hunk 16]
```diff
@@ -71,7 +71,7 @@
 if (!empty($wgEnableNukeExt)) {
 	$wgGroupPermissions['staff']['nuke'] = true;
 	$wgGroupPermissions['helper']['nuke'] = true;
-	$wgGroupPermissions['vstf']['nuke'] = true;
+	$wgGroupPermissions['soad']['nuke'] = true;
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php]:
```suggestion
	$wgGroupPermissions['soap']['nuke'] = true;
```

File: lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php [hunk 17]
```diff
@@ -166,7 +166,7 @@
 $wgGroupPermissions['*']['achievements-exempt'] = false;
 $wgGroupPermissions['helper']['achievements-exempt'] = true;
 $wgGroupPermissions['staff']['achievements-exempt'] = true;
-$wgGroupPermissions['vstf']['achievements-exempt'] = true;
+$wgGroupPermissions['soad']['achievements-exempt'] = true;
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php]:
```suggestion
$wgGroupPermissions['soap']['achievements-exempt'] = true;
```

File: lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php [hunk 18]
```diff
@@ -180,7 +180,7 @@
 $wgGroupPermissions['staff']['admindashboard'] = true;
 $wgGroupPermissions['sysop']['admindashboard'] = true;
 $wgGroupPermissions['threadmoderator']['admindashboard'] = true;
-$wgGroupPermissions['vstf']['admindashboard'] = true;
+$wgGroupPermissions['soad']['admindashboard'] = true;
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php]:
```suggestion
$wgGroupPermissions['soap']['admindashboard'] = true;
```

File: lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php [hunk 19]
```diff
@@ -367,7 +367,7 @@
 $wgGroupPermissions['util']['coppaimagereview'] = true;
 
 $wgGroupPermissions['util']['imagereview'] = true;
-$wgGroupPermissions['vstf']['imagereview'] = true;
+$wgGroupPermissions['soad']['imagereview'] = true;
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php]:
```suggestion
$wgGroupPermissions['soap']['imagereview'] = true;
```

File: lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php [hunk 20]
```diff
@@ -386,7 +386,7 @@
 
 
 $wgGroupPermissions['util']['promoteimagereview'] = true;
-$wgGroupPermissions['vstf']['promoteimagereview'] = true;
+$wgGroupPermissions['soad']['promoteimagereview'] = true;
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php]:
```suggestion
$wgGroupPermissions['soap']['promoteimagereview'] = true;
```

File: lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php [hunk 21]
```diff
@@ -408,7 +408,7 @@
 
 $wgGroupPermissions['staff']['multidelete'] = true;
 $wgGroupPermissions['helper']['multidelete'] = true;
-$wgGroupPermissions['vstf']['multidelete'] = true;
+$wgGroupPermissions['soad']['multidelete'] = true;
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php]:
```suggestion
$wgGroupPermissions['soap']['multidelete'] = true;
```

File: extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php [hunk 22]
```diff
@@ -2812,7 +2812,7 @@
 	'user-identity-box-group-staff' => 'Staff',
 	'user-identity-box-group-wiki-manager' => 'Gerentes Wiki',
 	'user-identity-box-group-helper' => 'Helper',
-	'user-identity-box-group-vstf' => 'VSTF',
+	'user-identity-box-group-soad' => 'SOAD',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php]:
```suggestion
	'user-identity-box-group-soap' => 'SOAP',
```

File: extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php [hunk 23]
```diff
@@ -2622,7 +2622,7 @@
 	'user-identity-box-group-staff' => 'Staff',
 	'user-identity-box-group-wiki-manager' => 'Menadżer Wiki',
 	'user-identity-box-group-helper' => 'Helper',
-	'user-identity-box-group-vstf' => 'VSTF',
+	'user-identity-box-group-soad' => 'SOAD',
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php]:
```suggestion
	'user-identity-box-group-soap' => 'SOAP',
```

File: extensions/wikia/UserProfilePageV3/UserTagsStrategy.class.php [hunk 24]
```diff
@@ -128,8 +128,8 @@ protected function getGlobalTags( $requested = 1 ) {
 				case 'helper':
 					$tags[] = wfMessage( 'user-identity-box-group-helper' )->escaped();
 					break;
-				case 'vstf':
-					$tags[] = wfMessage( 'user-identity-box-group-vstf' )->escaped();
+				case 'soad':
+					$tags[] = wfMessage( 'user-identity-box-group-soad' )->escaped();
```
Comment (Markus-Rost) [inline extensions/wikia/UserProfilePageV3/UserTagsStrategy.class.php]:
```suggestion
				case 'soap':
					$tags[] = wfMessage( 'user-identity-box-group-soap' )->escaped();
```

File: lib/Wikia/src/Service/User/Permissions/PermissionsConfiguration.php [hunk 25]
```diff
@@ -424,8 +424,8 @@ private function loadGroupsChangeableByGroups(
 		$this->groupsAddableByGroup['content-reviewer'] = [ 'content-reviewer' ];
 		$this->groupsRemovableByGroup['content-reviewer'] = [ 'content-reviewer' ];
 
-		$this->groupsSelfAddableByGroup['vstf'] = [ 'rollback', 'bot', 'sysop' ];
-		$this->groupsSelfRemovableByGroup['vstf'] = [ 'rollback', 'bot', 'sysop', 'bureaucrat' ];
+		$this->groupsSelfAddableByGroup['soad'] = [ 'rollback', 'bot', 'sysop' ];
+		$this->groupsSelfRemovableByGroup['soad'] = [ 'rollback', 'bot', 'sysop', 'bureaucrat' ];
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/PermissionsConfiguration.php]:
```suggestion
		$this->groupsSelfAddableByGroup['soap'] = [ 'rollback', 'bot', 'sysop' ];
		$this->groupsSelfRemovableByGroup['soap'] = [ 'rollback', 'bot', 'sysop', 'bureaucrat' ];
```

File: lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php [hunk 26]
```diff
@@ -35,11 +35,11 @@
...
-		$wgGroupPermissions['vstf']['abusefilter-modify'] = true;
-		$wgGroupPermissions['vstf']['abusefilter-modify-restricted'] = true;
-		$wgGroupPermissions['vstf']['abusefilter-revert'] = true;
-		$wgGroupPermissions['vstf']['abusefilter-log-detail'] = true;
-		$wgGroupPermissions['vstf']['abusefilter-private'] = true;
+		$wgGroupPermissions['soad']['abusefilter-modify'] = true;
+		$wgGroupPermissions['soad']['abusefilter-modify-restricted'] = true;
+		$wgGroupPermissions['soad']['abusefilter-revert'] = true;
+		$wgGroupPermissions['soad']['abusefilter-log-detail'] = true;
+		$wgGroupPermissions['soad']['abusefilter-private'] = true;
```
Comment (Markus-Rost) [inline lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php]:
```suggestion
		$wgGroupPermissions['soap']['abusefilter-modify'] = true;
		$wgGroupPermissions['soap']['abusefilter-modify-restricted'] = true;
		$wgGroupPermissions['soap']['abusefilter-revert'] = true;
		$wgGroupPermissions['soap']['abusefilter-log-detail'] = true;
		$wgGroupPermissions['soap']['abusefilter-private'] = true;
```

**Review by Markus-Rost — CHANGES_REQUESTED** [review]
Fix typo `SOAD` to `SOAP`

**Comment by maaartyyynaa** [comment]
Closing this messy one. Changes in https://github.com/Wikia/app/pull/18022 and https://github.com/Wikia/app/pull/18024

**Pull request closed without merging**